src/modules/RootPart.rb

```diff
@@ -483,11 +482,16 @@ def RunFSCKonJFS(mount_type, device, error_message)
     # @return [String] nil on success, error description on fail
     def MountPartition(mount_point, device, mount_type, fsopts = "")
       if mount_type == ""
-        # storage-ng
-        # FIXME: knowing the fstab string for the filesystem type should be
-        # responsibility of the upcoming Y2Storage::FilesystemType (to be
-        # added as part of the "yast-storage-ng as a libstorage wrapper" change)
-        mount_type = probed.partitions.with(name: device).filesystems.first.to_s
+
+        # FIXME
+        #
+        # Note that the code below will get passed unmodified fstab entries
+        # (like 'UUID=2f61fdb9-f82a-4052-8610-1eb090b82098') for device names
+        # and typically not match anything due to this.
+        #
+        # See also the comment in #update_staging_filesystem! below.
+        #
+        mount_type = fstype_for_device(probed, device) || ""
```

Comment:
Better than `fstab_string`, maybe it should be `mount_string` or something like that.